applications/sink/geode-sink/src/test/resources/logback-include.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
+
+	<appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d %5p %40.40c:%4L - %m%n</pattern>
+		</encoder>
+	</appender>
+
+	<appender name="delegate" class="org.springframework.geode.logging.slf4j.logback.DelegatingAppender"/>
+
+	<logger name="com.gemstone.gemfire" level="${spring.boot.data.gemfire.log.level:-WARN}"/>
```

Comment:
Is the property key something specific to the Geode starter or did we create it? We could instead just set a default level and then recommend the standard [Spring Boot method](https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#features.logging.log-levels) of adjusting the log level settings.

Reply:
This is probably cut and paste from an example somewhere.  The geode apps introduced a dependency on `spring-boot-data-gemfire` which came subsequent to these apps. Geode uses proprietary logging config, so it's got its own property now.  I noticed the pattern is different than the others. I don't have a strong opinion on this.